docker/Datadog.Trace.ClrProfiler.IntegrationTests.sh

```diff
@@ -3,6 +3,12 @@ set -euxo pipefail
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 
+$DIR/with-profiler-logs.bash \
```

Comment:
Thanks for the add here!

Reply:
Some of these tests were not designed for CI, so i'm going to skip them for now.